GameData/RP-0/changelog.cfg

```
@@ -1,4 +1,4 @@
-KERBALCHANGELOG
++KERBALCHANGELOG
 {
 	modName = RP-1
```

Auto-generated potential formatting fix:
```suggestion
+KERBALCHANGELOG
 {
{
	modName = RP-1
```